src/common.rs

```diff
@@ -9,6 +9,8 @@ use super::{
 
     Unalign, bitcast,
 };
+use std::mem;
+use std::ops::{Not, Neg, Add, Sub, Mul, Div, BitAnd, BitOr, BitXor, Shl, Shr};
```

Comment:
I would personally prefer this to just be `use std::ops;` and write `ops::Neg` below (etc.).